### DISCUSSION TITLE
Cant clone a repo (GitHub Desktop)

### DISCUSSION THREAD

**Discussion by xJ4111** [community/community#22303] [New to GitHub]
<p>Trying to clone a repo but keep getting errors, tried the following:</p>
<p>Increased post buffer</p>
<p>Depth 1 clone using Git Bash</p>
<p>Disabled firewall/anti-virus</p>
<p>Also tried downloading through ZIP and adding it to GitHub as a local repository but it wont recognise the repo (im not sure how it works/if im doing it right).</p>
<p>remote: Enumerating objects: 11, done. remote: Counting objects: 9% (1/11) remote: Counting objects: 18% (2/11) remote: Counting objects: 27% (3/11) remote: Counting objects: 36% (4/11) remote: Counting objects: 45% (5/11) remote: Counting objects: 54% (6/11) remote: Counting objects: 63% (7/11) remote: Counting objects: 72% (8/11) remote: Counting objects: 81% (9/11) remote: Counting objects: 90% (10/11) remote: Counting objects: 100% (11/11) remote: Counting objects: 100% (11/11), done. remote: Compressing objects: 9% (1/11) remote: Compressing objects: 18% (2/11) remote: Compressing objects: 27% (3/11) remote: Compressing objects: 36% (4/11) remote: Compressing objects: 45% (5/11) remote: Compressing objects: 54% (6/11) remote: Compressing objects: 63% (7/11) remote: Compressing objects: 72% (8/11) remote: Compressing objects: 81% (9/11) remote: Compressing objects: 90% (10/11) remote: Compressing objects: 100% (11/11) remote: Compressing objects: 100% (11/11), done. Receiving objects: 0% (1/4097) Receiving objects: 1% (41/4097) Receiving objects: 2% (82/4097) Receiving objects: 3% (123/4097), 540.01 KiB | 1.04 MiB/s Receiving objects: 3% (126/4097), 2.51 MiB | 2.49 MiB/s Receiving objects: 4% (164/4097), 5.07 MiB | 3.36 MiB/s Receiving objects: 4% (165/4097), 7.62 MiB | 3.78 MiB/s Receiving objects: 4% (173/4097), 12.52 MiB | 4.14 MiB/s Receiving objects: 5% (205/4097), 12.52 MiB | 4.14 MiB/s Receiving objects: 6% (246/4097), 12.52 MiB | 4.14 MiB/s Receiving objects: 6% (276/4097), 17.36 MiB | 4.31 MiB/s Receiving objects: 7% (287/4097), 19.64 MiB | 4.34 MiB/s Receiving objects: 7% (292/4097), 21.73 MiB | 4.68 MiB/s Receiving objects: 7% (306/4097), 26.96 MiB | 4.84 MiB/s Receiving objects: 7% (322/4097), 31.43 MiB | 4.75 MiB/s Receiving objects: 8% (328/4097), 33.36 MiB | 4.62 MiB/s Receiving objects: 8% (331/4097), 35.66 MiB | 4.62 MiB/s Receiving objects: 8% (342/4097), 40.40 MiB | 4.60 MiB/s Receiving objects: 8% (351/4097), 44.50 MiB | 4.50 MiB/s Receiving objects: 8% (363/4097), 48.82 MiB | 4.28 MiB/s Receiving objects: 9% (369/4097), 48.82 MiB | 4.28 MiB/s Receiving objects: 9% (372/4097), 52.82 MiB | 4.29 MiB/s Receiving objects: 9% (384/4097), 57.55 MiB | 4.28 MiB/s Receiving objects: 9% (395/4097), 62.34 MiB | 4.39 MiB/s Receiving objects: 9% (404/4097), 66.36 MiB | 4.36 MiB/s Receiving objects: 10% (410/4097), 66.36 MiB | 4.36 MiB/s Receiving objects: 10% (413/4097), 70.73 MiB | 4.39 MiB/s Receiving objects: 10% (422/4097), 75.30 MiB | 4.51 MiB/s Receiving objects: 10% (429/4097), 77.74 MiB | 4.47 MiB/s Receiving objects: 10% (438/4097), 81.68 MiB | 4.28 MiB/s Receiving objects: 10% (446/4097), 89.07 MiB | 4.56 MiB/s Receiving objects: 11% (451/4097), 91.11 MiB | 4.50 MiB/s Receiving objects: 11% (453/4097), 91.11 MiB | 4.50 MiB/s Receiving objects: 11% (464/4097), 98.66 MiB | 4.62 MiB/s Receiving objects: 11% (472/4097), 103.69 MiB | 4.87 MiB/s Receiving objects: 11% (479/4097), 105.59 MiB | 4.68 MiB/s Receiving objects: 11% (489/4097), 110.28 MiB | 4.70 MiB/s Receiving objects: 12% (492/4097), 112.95 MiB | 4.84 MiB/s Receiving objects: 12% (501/4097), 115.55 MiB | 4.88 MiB/s Receiving objects: 12% (512/4097), 121.93 MiB | 4.57 MiB/s Receiving objects: 12% (521/4097), 123.83 MiB | 4.46 MiB/s Receiving objects: 13% (533/4097), 128.71 MiB | 4.61 MiB/s fatal: early EOF fatal: the remote end hung up unexpectedly fatal: index-pack failed error: RPC failed; curl 56 Failure when receiving data from the peer</p>

**Comment by Wabri** [comment]
<p>Hi <a class="mention" href="/xj4111">@xj4111</a>,</p>
<p>Welcome to the GitHub Community forum!</p>
<p>This error is weird… Could you post the link of the repository? We can help in a better way if we can try to clone that!</p>
<p>-Gabriele-</p>
<blockquote>
<p><strong>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content!</strong></p>
</blockquote>

**Comment by mpboom** [comment]
<p>In addition to the URL of the repo <a class="mention" href="/wabri">@wabri</a> requested, it would also be helpful to know the exact command you used while cloning.</p>

**Comment by xJ4111** [comment, marked as answer]
<p>So it turns out it was a network side issue. I used a VPN and everything worked fine. Sorry for not updating the post sooner, thanks for the help though, appreciated.</p>
